pkg/remote/aws/test/ecr_repository_multiple/results.golden.json

```diff
@@ -0,0 +1,48 @@
+[
```

Comment:
It's a good practice to add within this directory the terraform file you used to generate theses resources.

If someone has to maintain your test in the future for example.

Reply:
right